SparkPost/2web2ui
# FE-1108 - Increase font size for help text on FileFieldWrapper component

[FE-1108](https://jira.int.messagesystems.com/browse/FE-1108)

### What Changed
- Increased font size from `"100"` to `"200"` for the `FileFieldWrapper` component's help text

### How To Test
- Go to `/lists/suppressions/create` and verify the font size is slightly bigger

### To Do
